packages/electron-chrome-extensions/src/browser/popup.ts

```diff
@@ -88,12 +88,24 @@ export class PopupView {
 
     if (this.destroyed) return
 
+    const hasChildNodes = await this.browserWindow!.webContents.executeJavaScript(
+      `((${() => {
+        return document.body.hasChildNodes();
+      }})())`
+    )
+
+    if (!hasChildNodes) {
+      this.destroy()
+      return
```

Comment:
My preference would be to figure out how Chromium is handling this rather than creating new behavior. There are likely subtle effects this can have which could break untested extensions.